components/reconciler/README.MD

```diff
@@ -4,11 +4,14 @@
 
 This module allows communication with the Reconciler's main component by generating the Mothership REST Client.
 
-The Mothership REST Client is generated from [this](https://raw.githubusercontent.com/kyma-incubator/reconciler/569388cbdde10092e5cdab25d5d87cf6a038a4d2/openapi/external_api.yaml) OpenAPI spec.
+The Mothership REST Client is generated
```

Comment:
I will use version from main after review, because this change is already in main